packages/react-components/src/styles/index.ts

```diff
@@ -32,7 +32,7 @@ function getContrast (uiHighlight: string | undefined): string {
 
   return brightness > BRIGHTNESS
     ? 'rgba(45, 43, 41, 0.875)'
-    : 'rgba(255, 253, 251, 0.875)';
+    : 'rgb(255, 255, 255)';
```

Comment:
We don't use pure-white anywhere. It creates too much contrast with the very dark backgrounds. (Hence the fade)